src/test/java/bio/terra/ChildContentHandlerTest.java

```diff
@@ -14,7 +14,7 @@ public void acceptEventTest() {
 
     try {
       StorageObjectData storageObjectData =
-          GsonWrapper.getInstance().fromJson(MOCK_EVENT_GZIP, StorageObjectData.class);
+          GsonWrapper.convertFromClass(MOCK_EVENT_GZIP, StorageObjectData.class);
       ChildContentHandler handler = new ChildContentHandler(storageObjectData);
       handler.setInputStream(MOCK_TGZ);
       handler.handleMediaType();
```

Comment:
see my comment on `FileTypeDetectorTest.java` - this needs a `fail` inside its catch clause, and it needs an assertion that it actually processed the expected number of archive entries.

Reply:
Added `fail` and checks for the expected number of files processed.